src/root-command.ts

```diff
@@ -25,19 +28,38 @@ export default abstract class extends Command {
   }
 
   private _mesg: Application | null = null
+  private _serviceSDK: any
+
+  get engineEndpoint(): string {
+    const host = process.env.DOCKER_HOST
+      ? new URL(process.env.DOCKER_HOST).hostname
+      : 'localhost'
+    return `${host}:50052`
+  }
 
   get mesg() {
     if (!this._mesg) {
-      const host = process.env.DOCKER_HOST
-        ? new URL(process.env.DOCKER_HOST).hostname
-        : 'localhost'
-      this._mesg = application({
-        endpoint: `${host}:50052`
-      })
+      this._mesg = application({endpoint: this.engineEndpoint})
     }
     return this._mesg
   }
 
+  get serviceSDK() {
```

Comment:
would rename with API instead of SDK but not blocking

Reply:
done